src/gsUtils/gsUtils.h

```diff
@@ -138,6 +138,36 @@ inline void string_replace(std::string& str,
     }
 }
 
+/// \brief Splits a string \a str into substrings with \a ch as separator.
```

Comment:
You should mention that empty "items" are ignored

Reply:
Yep, used this as helperfunction for [`gsFileManager::makeRelative`](https://github.com/gismo/gismo/blob/dd34709aeff7d771093ccd7bdd6ef29bb876392e/src/gsIO/gsFileManager.cpp#L469). Therefore I ignored empty ones complete - there exists no empty subpaths, neither under Windows nor under Unix. But as you can see here: https://github.com/gismo/gismo/blob/dd34709aeff7d771093ccd7bdd6ef29bb876392e/src/gsIO/gsFileManager.cpp#L485-L508
I implemented a O(n) way without splitting and didn't thought more an this helper function - just kept it in if somebody else could need it - we can throw it away if you want.